reactor-netty-http/src/test/java/reactor/netty/http/server/ConnectionInfoTests.java

```diff
@@ -713,6 +758,13 @@ void forwardedForIpV6() {
 				});
 	}
 
+	private void testClientRequest(Consumer<HttpHeaders> clientRequestHeadersConsumer,
```

Comment:
This new method is introduced just to invoke an existing method with the proper forwarded handler. Why don't we invoke the existing method with the proper forwarded handler as it is done in some of the tests?

Reply:
This new method has been introduced in order to reduce the changes made for all other tests.

Example: let's take the xForwardedFor, line 241, which is invoking the new introduced method:
```java
	@ParameterizedTest(name = "{displayName}({arguments})")
	@ValueSource(booleans = {true, false})
	void xForwardedFor(boolean useCustomForwardedHandler) {
		testClientRequest(
				clientRequestHeaders -> clientRequestHeaders.add("X-Forwarded-For",
						"[1abc:2abc:3abc::5ABC:6abc]:8080, 192.168.0.1"),
				serverRequest -> {
					Assertions.assertThat(serverRequest.remoteAddress().getHostString()).isEqualTo("1abc:2abc:3abc:0:0:0:5abc:6abc");
					Assertions.assertThat(serverRequest.remoteAddress().getPort()).isEqualTo(8080);
				},
				useCustomForwardedHandler);
	}
```
without the new method introduced line 761, the above method would be:
```
	@ParameterizedTest(name = "{displayName}({arguments})")
	@ValueSource(booleans = {true, false})
	void xForwardedFor(boolean useCustomForwardedHandler) {
		testClientRequest(
				clientRequestHeaders -> clientRequestHeaders.add("X-Forwarded-For",
						"[1abc:2abc:3abc::5ABC:6abc]:8080, 192.168.0.1"),
				serverRequest -> {
					Assertions.assertThat(serverRequest.remoteAddress().getHostString()).isEqualTo("1abc:2abc:3abc:0:0:0:5abc:6abc");
					Assertions.assertThat(serverRequest.remoteAddress().getPort()).isEqualTo(8080);
				},
				getForwardedHandler(useCustomForwardedHandler),
				Function.identity(),
				Function.identity(),
				false);
	}
```

So, with the new introduced method, the modifications done in all other xForwarded* are shorted.
But Ok,  I will revert and remove the new introduced method.

Reply:
ok then I'll leave you to decide

Reply:
I would prefer to keep the existing method, in order to reduce the code diff.